content/en/how-to-guides/configuration-management/_index.adoc

```diff
@@ -11,50 +11,5 @@ description: >
 These guidelines assume that you're familiar with the GitHub workflow.
 {{% /pageinfo %}}
 
-We use a https://www.writethedocs.org/videos/portland/2019/lessons-learned-in-a-year-of-docs-driven-development-jessica-parsons/[docs-driven development] approach, using the GitHub workflow, to facilitate configuration management for Project Tetra. The documentation for this project is maintained in the https://github.com/tetrabiodistributed/project-tetra-docs[Project Tetra docs repo] with the project source code maintained in the https://github.com/helpfulengineering/project-tetra[Project Tetra repo].
+The following pages are intended for internal use by the Project Tetra team.
```

Comment:
Leave a note that these documentations pages are works in progress at the top of the page along with date and time last updated.

Reply:
No need to manually stamp date/time; Hugo automatically adds a _Last modified_ date/time stamps with a along with a reference to the last commit made for the page being viewed at the bottom of the page.